aiobotocore/endpoint.py

```diff
@@ -44,9 +48,45 @@ def __getattr__(self, item):
 
     def __dir__(self):
         attrs = dir(self.__content)
-        attrs.append('close')
+        attrs += ['read_all', 'text', 'json' 'close']
         return attrs
 
+    def __del__(self):
+        self.close()
+
+    @asyncio.coroutine
+    def read_all(self):
+        """
```

Comment:
removed!  I actually realized that read_all wasn't needed as I can now use "async with" with the aenter/aexit methods I added to get at the response object and call on it directly